warehouse/ingest-core/src/test/java/datawave/ingest/data/config/ingest/WhindexFieldIngestHelperTest.java

```diff
@@ -0,0 +1,259 @@
+package datawave.ingest.data.config.ingest;
+
+import java.util.Arrays;
+
+import com.google.common.collect.HashMultimap;
+import datawave.ingest.data.config.NormalizedContentInterface;
+import datawave.ingest.data.config.NormalizedFieldAndValue;
+import org.apache.hadoop.conf.Configuration;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.LinkedListMultimap;
+import com.google.common.collect.Multimap;
+
+import datawave.ingest.data.Type;
+
+class WhindexFieldIngestHelperTest {
+
+    /*
+     * Tests that a configuration without a <type>.rules property does not have whindex field definitions.
```

Comment:
The documentation is out of date. Whindex field configurations are now supplied via properties with the  `<type>.whindex.rules` prefix.